modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss

```diff
@@ -1,16 +1,606 @@
+
+$small-font-size: 0.875rem !default;
+
+$text-muted: var(--text-muted, $gray-500) !default;
+
+$blockquote-small-color: var(--blockquote-small-color, $gray-600);
+$blockquote-small-font-size: var(--blockquote-small-font-size, $small-font-size);
+$blockquote-font-size: var(--blockquote-font-size, $font-size-base * 1.25);
+
+$hr-border-color: var(--hr-border-color, rgba($black, 0.1)) !default;
+$hr-border-width: var(--hr-border-width, $border-width) !default;
+$hr-margin-y: var(--hr-margin-y, $spacer) !default;
+
+$mark-padding: 0.2em !default;
+
+$dt-font-weight: $font-weight-bold !default;
+
+$code-color: $pink !default;
+$code-font-size: 87.5% !default;
+
+$kbd-bg: $gray-900 !default;
+$kbd-color: $white !default;
+$kbd-font-size: $code-font-size !default;
+$kbd-padding-x: 0.4rem !default;
+$kbd-padding-y: 0.2rem !default;
+$kbd-box-shadow: inset 0 -0.1rem 0 rgba($black, 0.25) !default;
+
+$nested-kbd-font-weight: $font-weight-bold !default;
+
+$pre-color: $gray-900 !default;
+$pre-scrollable-max-height: 340px !default;
+
+$list-inline-padding: 0.5rem !default;
+
+$mark-bg: #fcf8e3 !default;
+
+// Button
+
+$c-button-base: () !default;
+$c-button-base: map-merge(
+	(
+		cursor: $link-cursor,
+		-webkit-appearance: button,
+	),
+	$c-button-base
+);
+
+$close-text-shadow: none !default;
+
+// Buttons + Forms
+
+$label-margin-bottom: 0.5rem !default;
+
+$input-btn-font-family: null !default;
+$input-btn-font-size: $font-size-base !default;
+$input-btn-line-height: $line-height-base !default;
+$input-btn-padding-x: 0.75rem !default;
+$input-btn-padding-y: 0.375rem !default;
+
+$input-btn-focus-width: 0.2rem !default;
+
+$input-btn-focus-color: rgba($component-active-bg, 0.25) !default;
+$input-btn-focus-box-shadow: 0 0 0 $input-btn-focus-width $input-btn-focus-color !default;
+
+$input-btn-font-size-sm: $font-size-sm !default;
+$input-btn-line-height-sm: $line-height-sm !default;
+$input-btn-padding-x-sm: 0.5rem !default;
+$input-btn-padding-y-sm: 0.25rem !default;
+
+$input-btn-font-size-lg: $font-size-lg !default;
+$input-btn-line-height-lg: $line-height-lg !default;
+$input-btn-padding-x-lg: 1rem !default;
+$input-btn-padding-y-lg: 0.5rem !default;
+
+$input-btn-border-width: $border-width !default;
+
+// Print
+
+$print-body-min-width: map-get($grid-breakpoints, 'lg') !default;
+$print-page-size: a3 !default;
+
+// Start Theme
+
 $border-color: #f1f2f5;
 $bright-color: #1865fb;
 $complementary-color: #30313f;
 $dark-color: #272833;
 $input-bg: #f1f2f5;
 $light-color: #e7e7ed;
 
-$lead-font-weight: 400;
-$small-font-size: 0.875rem;
+$portlet-decorate-bg: #fff;
+$portlet-decorate-border: 1px solid $light-color;
 
-$btn-outline-secondary: (
-	color: #6b6c7e,
+$btn-primary: () !default;
+$btn-primary: map-deep-merge(
+	(
+		background-color: var(--btn-primary-background-color, $primary),
+		border-color: var(--btn-primary-border-color, $primary),
+		box-shadow: $c-unset,
+		color: var(--btn-primary-color, color-yiq($primary)),
```

Comment:
In these cases we shouldn't use `--primary` token instead of creating a component-specific one, it seems a bit inconsistent, for example in theory `btn-primary` would follow the colors of `primary`, as well as subsequent ones, with would that allow you to change the color of `btn-primary` without following the global color of `--primary`, in which case wouldn't it be a bit contradictory? And if the user wants a different color for the button, wouldn't it be a new variation?